vhdbuilder/packer/windows/windows_settings.json

```diff
@@ -39,14 +39,28 @@
       "os_disk_size": "35",
       "base_image_sku": "23h2-datacenter-core",
       "windows_image_name": "windows-23H2",
-      "base_image_version": "25398.1425.250210",
+      "base_image_version": "",
       "patches_to_apply": []
     },
     "23H2-gen2": {
       "os_disk_size": "35",
       "base_image_sku": "23h2-datacenter-core-g2",
       "windows_image_name": "windows-23H2",
-      "base_image_version": "25398.1425.250210",
+      "base_image_version": "",
```

Comment:
Same comment as above - we shouldn't be changing the base version of 23H2 in this PR.